algos/demux/demux.rst~

```diff
@@ -0,0 +1,78 @@
+.. _demux:
+
+Muliplexer/Demultiplexer
+########################
```

Comment:
This looks like a duplicate of the previous file ?

Reply:
oh, emacs backup, sorry will remove.